tests/test_streaming_download_manager.py

```diff
@@ -425,6 +425,10 @@ def test_streaming_dl_manager_extract_all_supported_single_file_compression_type
         ("zip://train-00000.json.gz::https://foo.bar/data.zip", "gzip"),
         ("https://foo.bar/train.json.gz?dl=1", "gzip"),
         ("http://opus.nlpl.eu/download.php?f=Bianet/v1/moses/en-ku.txt.zip", "zip"),
```

Comment:
> Great! For the tests, I think we should also add some URLs in the form: `http://ufal.ms.mff.cuni.cz/umc/005-en-ur/download.php?f=umc005-corpus.zip` to be sure they are still correctly detected.

@severo please note that test case was already implemented before this PR (see line above). ;)

Reply:
oh right 🤦‍♂️